src/data/constants.js

```diff
@@ -19,3 +19,6 @@ export const NR1_BASE_URL = 'https://staging-one.newrelic.com';
 
 export const NR1_PACK_DETAILS_NERDLET =
   'catalog-pack-details.catalog-pack-contents';
+
+export const REPO_URL =
+  'https://github.com/newrelic/newrelic-observability-packs';
```

Comment:
Nitpick: `REPO_URL` is a little ambiguous (sorry, I should have provided a better suggestion). Since this is the URL for the Observability Packs / Quickstarts and not the current repository (developer-website), we should probably update this accordingly:
```suggestion
export const QUICKSTARTS_REPO =
  'https://github.com/newrelic/newrelic-observability-packs';
```